repo/meta/index.json

```diff
@@ -112,5 +112,5 @@
       }
     }
   ],
-  "version":"0.1.0-alpha"
+  "version":"1.0.0-rc1"
```

Comment:
It looks like there's a repo/meta/version.json file that's still 0.1.0.  Is that a mistake?